package.json

```diff
@@ -137,11 +137,11 @@
     "emoji-regex": "^10.2.1",
     "es6-promise": "^4.2.8",
     "esbuild": "^0.10.x",
-    "eslint": "^8.31.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-mocha-no-only": "^1.1.1",
```

Comment:
unfortunately I was unable to reuse this in eslint v9 (this package has not been updated in 4 yrs and does not support the new eslint syntax)

Reply:
We could add a `no-restricted-syntax` rule that prohibits `it.only`, `test.only` and `describe.only`.

Reply:
I just wrote the fix to the plugin https://github.com/lewazo/eslint-mocha-no-only/pull/6